### DISCUSSION TITLE
PC 버전 활성화

### DISCUSSION THREAD

**Discussion by Astoration** [naverz/zepeto-studio-kor#888] [5. Q&A]
PC버전 활성화는 어떻게 하는건가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 PC 버전 플레이를 말씀하시는 거라면
현재는 공식맵 및 빌드잇 맵만 PC버전 플레이가 가능합니다.
유니티 월드도 PC 버전 플레이를 지원할 수 있게 준비 중입니다.
감사합니다.
